<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmergencyBroadcast</class>
 <widget class="QWidget" name="EmergencyBroadcast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 20px, &quot;SimHei&quot;;
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;
}</string>
  </property>
  <widget class="QPushButton" name="btn0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 30px</string>
   </property>
   <property name="text">
    <string>无紧急广播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 30px</string>
   </property>
   <property name="text">
    <string>故障站台清客</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 30px</string>
   </property>
   <property name="text">
    <string>清客警示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 30px</string>
   </property>
   <property name="text">
    <string>救援站台清客</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 30px</string>
   </property>
   <property name="text">
    <string>区间清客</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>180</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 30px</string>
   </property>
   <property name="text">
    <string>其他站台清客</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 30px</string>
   </property>
   <property name="text">
    <string>大小交路转换</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 30px</string>
   </property>
   <property name="text">
    <string>故障临时停车</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Back">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>570</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cease">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>570</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>停止广播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>290</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 30px</string>
   </property>
   <property name="text">
    <string>其他临时停车</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
